docs/architecture/microservices/net-core-net-framework-containers/container-framework-choice-factors.md

```diff
@@ -11,17 +11,17 @@ The following decision table summarizes whether to use .NET Framework or .NET Co
 > Your development machines will run one Docker host, either Linux or Windows. Related microservices that you want to run and test together in one solution will all need to run on the same container platform.
 
 | Architecture / App Type | Linux containers | Windows Containers |
-|-------------------------|------------------|--------------------|
+|-------------------------|:----------------:|:------------------:|
 | Microservices on containers | .NET Core | .NET Core |
-| Monolithic app | .NET Core | .NET Framework <br/> .NET Core |
+| Monolithic app | .NET Core | .NET Framework <br> .NET Core |
 | Best-in-class performance and scalability | .NET Core | .NET Core |
 | Windows Server legacy app ("brown-field") migration to containers | -- | .NET Framework |
 | New container-based development ("green-field") | .NET Core | .NET Core |
-| ASP.NET Core | .NET Core | .NET Core (recommended) <br/> .NET Framework |
+| ASP.NET Core | .NET Core | .NET Core (recommended) <br> .NET Framework |
 | ASP.NET 4 (MVC 5, Web API 2, and Web Forms) | -- | .NET Framework |
-| SignalR services | .NET Core 2.1 or higher version | .NET Framework <br/> .NET Core 2.1 or higher version |
-| WCF, WF, and other legacy frameworks | WCF in .NET Core (client library only) | .NET Framework <br/> WCF in .NET Core (client library only) |
-| Consumption of Azure services | .NET Core <br/> (eventually all Azure services will provide client SDKs for .NET Core) | .NET Framework <br/> .NET Core <br/> (eventually all Azure services will provide client SDKs for .NET Core) |
+| SignalR services | .NET Core 2.1 or higher version | .NET Framework <br> .NET Core 2.1 or higher version |
+| WCF, WF, and other legacy frameworks | WCF in .NET Core (client library only) | .NET Framework <br> WCF in .NET Core (client library only) |
+| Consumption of Azure services | .NET Core <br> (eventually most Azure services will provide client SDKs for .NET Core) | .NET Framework <br> .NET Core <br> (eventually most Azure services will provide client SDKs for .NET Core) |
```

Comment:
and self closing br tags cause issues too? That's usually how we use those tags in docs?

Reply:
Nope, not really. it was actually `<br>` that caused problems with the generator (not anymore).

The point is that I stumped with another table that had `<br>` (that's when I discovered the generator wasn't handling them correctly) and, out of curiosity, researched a little and found that `<br>` is an **html** tag and `<br/>` is an **xhtml** tag. Then checked that docs emits  html:

![image](https://user-images.githubusercontent.com/7357806/74934756-cd464100-53de-11ea-8bd0-0fc58359e2a9.png)

So that's why I switched to `<br>`, but nothing else.

Can revert that if you prefer, anyway.

Reply:
Ah no preference, just curiosity.